packages/host/src/components/AppDrawer/AppDrawer.test.tsx

```diff
@@ -83,18 +77,16 @@ describe('AppDrawer', () => {
       name: /close the menu/i,
     });
 
-    act(() => {
-      userEvent.click(button);
-    });
+    fireEvent.click(button);
 
-    waitFor(() => {
+    await waitFor(() => {
       expect(getByText(/dashboard/i)).not.toBeVisible();
       expect(getByText(/distribution/i)).not.toBeVisible();
       expect(getByText(/suppliers/i)).not.toBeVisible();
-      expect(getByText(/stock/i)).not.toBeVisible();
-      expect(getByText(/tools/i)).not.toBeVisible();
+      expect(getByText(/stock$/i)).not.toBeVisible();
+      // expect(getByText(/tools/i)).not.toBeVisible();
```

Comment:
these were failing - but not noticed because we weren't waiting for them. Adding the `await` fixes an issue with the test runner, and highlighted the failing tests

Reply:
There is a linter rule "no-floating-promises" which looks for these kind of bugs :)

Reply:
thanks! wasn't aware of that one.

We are using `plugin:@typescript-eslint/recommended` which includes the [recommended rules](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/eslint-plugin/docs/rules) which in turn means we are already using that rule.. and it didn't pick up this particular problem. The issue with the test is that it is checking for an element which won't exist, because we removed that menu item!